DEPLOYING-FOR-TESTING.md

```diff
@@ -0,0 +1,125 @@
+# Deploying the operator for testing
+
+This section describess testing your local changes of the custom domains operator. 
+First, create a cluster in stage where you plan to deploy the custom domains operator.
+
+
+### Building and pushing image in Quay
+
+To deploy the operator from your branch, you have to create an image in Quay. First, log in to quay and create a folder where you will push the image. In this example I will build the image in quay and push it to my quay account **aliceh** to a folder named **cdo**  with a tag **0**.
```

Comment:
```suggestion
To deploy the operator from your branch, you have to create an image in Quay under your personal account. For example, this will build the image in quay and push it to my personalquay account **aliceh** to a repository named **cdo**  with a tag **0**.
```